File: app/src/main/java/com/mxt/anitrend/model/entity/base/NotificationHistory.java
package com.mxt.anitrend.model.entity.base;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;

@Entity
public class NotificationHistory {

    @Id(assignable = true)
    private long id;
    private boolean read;

    public NotificationHistory() {

    }

    public NotificationHistory(long id) {
        this.id = id;
    }

    public long getId() {
        return id;
    }

    public boolean isRead() {
        return read;
    }

    public void setId(long id) {
        this.id = id;
    }

    public void setRead(boolean read) {
        this.read = read;
    }
}

File: app/src/main/java/com/mxt/anitrend/model/entity/base/NotificationHistory.kt
package com.mxt.anitrend.model.entity.base

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
class NotificationHistory() {

    @Id(assignable = true)
    var id: Long = 0
    var read: Boolean = false

    constructor(id: Long) : this() {
        this.id = id
    }
}